### ISSUE TITLE
improvement of sampling pipeline

### ISSUE THREAD

**Issue by JanHasenauer** [ICB-DCM/PESTO#122]
Sampling currently requires quite a bit of manual work, e.g. shortening of chains, ... . I think it would be great to have a routine which performs multiple sampling runs, automatically removes the burn-in and checks for reproducibility. This could be implemented in getParameterSamples.

**Comment by JanHasenauer** [comment]
I agree that for large problems, one usually needs more computational resources. For smaller problems, this is however not the case. One application example would be Elba's model for hematopoiesis. I think it would be really nice to have a function getMultipleParameterSamples (similar to what we have for optimisation), which also performs the analysis. Even if persons cannot use the function write away in a server setting, they would be able to see what steps have to be taken. For many users this is not clear.

**Comment by BBallnus** [comment]
What do you think about an in-depth example covering such a functionality by calling multipe instances of getParameterSamples() followed by the application of the analysis pipeline? Such piece of code could be used by everyone interested in robust sampling assessment. With a function as getMultipleParameterSamples, I currently see the danger missleading unexperienced users.

**Comment by JanHasenauer** [comment]
An example is good, but in my opinion not sufficient. One never knows whether a user looks at all the examples.

What danger do you see regarding: "With a function as getMultipleParameterSamples, I currently see the danger missleading unexperienced users."?

Apparently, no method implemented in PESTO will always work. This I wouldn't see as a problem.
